tsconfig.base.json

```diff
@@ -5,14 +5,14 @@
     "esModuleInterop": true,
     "jsx": "react-jsx",
     "lib": ["dom", "dom.iterable", "esnext"],
-    "module": "esnext",
+    "module": "ESNext",
```

Comment:
let's make this es2020 as well. So we can manually change if needed with version increase.